source/agora/protocol/Cache.d

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
```

Comment:
You should merge the first two commits into one. We shouldn't add code in one commit that is then removed in the second commit. You can use an interactive rebase to "squash" the two commits into one.

Reply:
Thank you.